models/package.go

```diff
@@ -0,0 +1,382 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
```

Comment:
How about move this file into `models/package/package.go` so that more package related models could be added easier.

Reply:
done